### ISSUE TITLE
Laser Alignment for Gerald

### ISSUE THREAD

**Issue by DaveBenji** [DoESLiverpool/somebody-should#1883]
There is a problem with Gerald, the cut work frame is larger than the drawing dimensions and out of line by 5 mm to the left on the x axis

**Comment by johnmckerrell** [comment]
@DoESLiverpool/laser-maintenance FYI

**Comment by DaveBenji** [comment]
Is there an update?

**Comment by magman2112** [comment]
This has been corrected now.

The X-Axis was measuring 619mm on a 600mm cut, so I adjusted the settings on the controller SD card to correct for this. I also adjusted the Y-Axis at the same time, as this was out also (but not so much 503mm in 500mm).

Not sure why this has got so far out, as this adjustment should only be made in the firmware configuration file. Will do period checks to see if this re-occurs. 

It might be related to the replacment drive belt in #1869. So we should also complete the order for the correct replacement belt for this issue as well.
